src/EditorFeatures/Core.Wpf/SignatureHelp/Controller.Session_ComputeModel.cs

```diff
@@ -171,6 +192,10 @@ private static SignatureHelpItem GetSelectedItem(Model currentModel, SignatureHe
                 return lastSelectionOrDefault;
             }
 
+            private static bool WillChangeDisplay(Model currentModel, ISignatureHelpProvider provider, SignatureHelpItems items)
+                => currentModel.Provider != provider ||
+                   !DisplayPartsMatch(currentModel.SelectedItem, items.Items[items.SelectedItemIndex.GetValueOrDefault()]);
```

Comment:
this lines makes me feel oogy (the GetValueOrDefault()).  I'm not sure if it's right or wrong, or if it will cause problems.  it just makes me uneasy :)

Reply:
I wouldn't want you to feel oogy. Would you prefer `items.SelectedItemIndex ?? 0` to make it more explicit? Or explode this out to multiple checks of provider, SelectedItemIndex.HasValue, etc.?